manifests/plugin/sensors.pp

```diff
@@ -1,21 +1,27 @@
 # https://collectd.org/wiki/index.php/Plugin:Sensors
 class collectd::plugin::sensors (
-  $ensure           = 'present',
-  $manage_package   = undef,
-  $sensorconfigfile = undef,
-  $sensor           = undef,
-  $ignoreselected   = undef,
-  $interval         = undef,
+  $ensure                  = 'present',
+  $manage_package          = undef,
+  $sensorconfigfile        = undef,
+  $sensor                  = undef,
+  $ignoreselected          = undef,
+  $interval                = undef,
+  $package_install_options = $collectd::package_install_options,
 ) {
 
   include ::collectd
 
   $_manage_package = pick($manage_package, $::collectd::manage_package)
 
+  if $package_install_options != undef {
+    validate_array($package_install_options)
```

Comment:
This should work:

```puppet
Optional[Array[String]] $package_install_options = undef
```

our time is limited, we will refactore the whole code somewhere in the future. We would like to already use the new datatypes for all new pull requests.